connect_four/board.rb

```diff
@@ -0,0 +1,130 @@
+class Board
+  attr_accessor :board
```

Comment:
I'm interested to see what the attr_accessor :board does. only one way to find out! onward!